### ISSUE TITLE
runtime error at `webview.queryError.readwrite`

### ISSUE THREAD

**Issue by tsy-pavel** [yy0931/sqlite3-editor#28]
foreign key mismatch - "schedules" referencing "masters"
Query: DELETE FROM "masters" WHERE rowid = %3F
Params: [1]
But "schedules" table is empty.

**Comment by tsy-pavel** [comment]
before the last update all was working...

**Comment by yy0931** [comment]
I believe it is because I enabled foreign key checks (`PRAGMA foreign_keys`) in v1.0.91. Does executing `PRAGMA foreign_keys = OFF;` in the query editor fix your problem? The statement disables foreign key checks in the current connection.

I have previously added a message that explains the above for the `FOREIGN KEY constraint failed` error, but I overlooked the `foreign key mismatch` error. I'll fix that in the next version.
